tests/virtualization/universal/windows.pm

```diff
@@ -26,11 +26,15 @@ sub remove_guest {
     }
 }
 
+sub remove_guests() {
+    remove_guest $_ foreach (keys %virt_autotest::common::imports);
+}
```

Comment:
Two things:
 * Why is this here and not in `virt_autotest::utils`?
 * Whys this even exists? We have ton of functions nobody barely uses.

Reply:
I was using it twice within the same module and line 37 looks much slimmer with this.
IMHO this function is really too small to justify a lib function.